.eslintrc.cjs

```diff
@@ -27,6 +27,25 @@ const deprecatedTerminology = Object.fromEntries(
   ]),
 );
 
+/**
+ * Rules for code that crosses an asyncFlow membrane.
+ */
+const resumable = [
```

Comment:
The selectors seem to pick out code that is *not* resumable. Is it conventional to name rules this way?

Reply:
The thought was that resumable has to satisfy these rules. I think that's conventional. E.g. `airbnb-style` rule set defines a set of rules in order to confirm to Airbnb style.